janitor/functions/expand_grid.py

```diff
@@ -109,6 +96,19 @@ def expand_grid(
 
         check("df_key", df_key, [str])
 
+    if not others and (df is not None):
+        return df
+
+    if not others:
+        return None
```

Comment:
Can we update the docstring (the `Returns:` section) to specify when None would be returned instead of a dataframe? I missed that this was a possibility before.

(And a little curious why we're returning None instead of an empty dataframe here) The latter would be useful for ensuring method chaining always works.

Reply:
You can build a dataframe with a dictionary - `expand_grid` is not limited to dataframes: 

```py
data = {"x": [1, 2, 3], "y": [1, 2]}
jn.expand_grid(others=data)
   x  y
   0  0
0  1  1
1  1  2
2  2  1
3  2  2
4  3  1
5  3  2
```

line 101 ensures chaining always works, as long as `df` is provided: 

```py
    if not others and (df is not None):
        return df
```